components/dropdown/dropdown-opener-mixin.js

```diff
@@ -73,6 +127,43 @@ export const DropdownOpenerMixin = superclass => class extends superclass {
 		this.addEventListener('d2l-dropdown-close', this.__onClosed);
 	}
 
+	updated(changedProperties) {
+		super.updated(changedProperties);
+		if (!this.openOnHover) return;
```

Comment:
A consequence of doing this in `updated` is that if the dropdown is opened, subsequently disconnected and then reconnected to the DOM (ex. drag & drop), the handlers would be removed (above in `disconnectedCallback`) but they would not be re-added unless `updated` is called for some other reason.  This is perhaps an unlikely scenario though - it might require hovering to open the dropdown and then switching to keyboard to perform such an action.  Another way would be if the dropdown content contained an action that caused this disconnect/reconnect scenario.  Strange, but stranger things have been done.  

Is it necessary to wire-up directly to the opener and content elements - can we just wire up to `this`? In such a case, we avoid accessing the `this.shadowRoot` and the wire-up can be done in `connectedCallback`.